yt_dlp/extractor/line.py

```diff
@@ -82,14 +83,16 @@ def _real_extract(self, url):
 
         # like_count requires an additional API request https://tv.line.me/api/likeit/getCount
 
+        thumbnail = try_get(video_info, lambda x: x['meta']['cover']['source']) or []
+        thumbnail = thumbnail.split('?')[0]#Full Size Thumbnail
```

Comment:
this is wrong.
why are u using `or []`?
This will set the thumbnail as a list when try_get returns none or '' and then the .split will throw error.
Also is the splitting really needed?

Reply:
I'm not familiar with python should it be "" instead of the []
The split is needed to get he full size thumbnail, otherwise it is scaled down to what they put in the query string.

Reply:
```suggestion
        thumbnail = url_or_none(try_get(video_info, lambda x: x['meta']['cover']['source']))
        if thumbnail:
            thumbnail = thumbnail.split('?')[0]
```
Import url_or_none from utils if not already imported.
This is assuming thumbnail and thumbnails both can be returned at same time.
If not then move the thumbnails code here and then append this thumbnail to it aswell.

Reply:
url_or_none isn't imported, I suggested it to be added.
The original thumbnail code is pointing to images that are not designed to be thumbnails.